views/index.hbs

```diff
@@ -0,0 +1,7 @@
+<h2>Homepage</h2>
+
+<form action="/artist-search" method="GET">
+<label>Search</label>
+<input type="text" name="artist">
+<button type="submit">Search</button>
+</form>
```

Comment:
Try to always format your files (indentation) ;)  easier to read  for you and the person who is reviewing your code